en/docs/security-guidelines/secure-engineering-guidelines/dynamic-analysis-with-owasp-zap.md

```diff
@@ -16,199 +16,199 @@ This document provides details of all necessary steps for configuring the OWASP
 
 
 ### Fine Tune ZAP Tool with Pre-Configured Policy 
-The ZAP tool should be fine tuned before running a scan for obtaining better results. For this, you can download the WSO2 policy file for ZAP[^2] which contains the settings to fine tune ZAP. 
+The ZAP tool should be fine-tuned before running a scan for obtaining better results. For this, you can download the WSO2 policy file for ZAP[^2] which contains the settings to fine-tune ZAP. 
 
 Go to **Analyze** &rarr; **Scan Policy** Manager in ZAP.
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-01.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-01.png)
 
-In the **Scan Policy Manager** window, click on **Import**.
+In the **Scan Policy Manager** window, click **Import**.
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-02.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-02.png)
 
 Browse and select the WSO2 Policy file you downloaded.
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-03.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-03.png)
 
 Since the policy file is imported correctly, you can use this later when you run the spider and the scan. 
 
 
 ### Configuring ZAP Proxy to Trace Browser Traffic
-Rather than providing the URL of the WSO2 server and attacking the URL with ZAP, it is much more effective if we record the UI actions we do on the WSO2 server and let ZAP tool capture the traffic which can then be used for performing attacks. 
+Rather than providing the URL of the WSO2 server and attacking the URL with ZAP, it is much more effective if we record the UI actions we do on the WSO2 server and let the ZAP tool capture the traffic which can then be used for performing attacks. 
 
-Go to **Tools** &rarr; **Options** &rarr; **Local Proxy** and set the hostname/ ip address and the port number for the proxy. *(In this example, the port is set to 7777 which is selected randomly)*
+Go to **Tools** &rarr; **Options** &rarr; **Local Proxy** and set the hostname/IP address and the port number for the proxy. *(In this example, the port is set to 7777 which is selected randomly)*
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-04.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-04.png)
 
-Now the ZAP tool is ready to capture the traffic going through the above set port number. Next step is to configure the browser to send traffic through this port number so that the ZAP tool can trace them.
+Now the ZAP tool is ready to capture the traffic going through the above set port number. The next step is to configure the browser to send traffic through this port number so that the ZAP tool can trace them.
```

Comment:
```suggestion
Now the ZAP tool is ready to capture the traffic going through the port (set above). The next step is to configure the browser to send traffic through this port number so that the ZAP tool can trace them.
```